bentoml/spacy.py

```diff
@@ -207,22 +207,28 @@ def save(
     return _model.tag
 
 
-# TODO: save local projects
-# bentoml.spacy.projects('save', /path/to/local/spacy/project)
 @inject
 def projects(
     save_name: str,
     tasks: str,
     name: t.Optional[str] = None,
-    repo_or_store: str = DEFAULT_SPACY_PROJECTS_REPO,
+    repo_or_store: t.Optional[str] = None,
     remotes_config: t.Optional[t.Dict[str, t.Dict[str, str]]] = None,
     *,
-    branch: str = DEFAULT_SPACY_PROJECTS_BRANCH,
+    branch: t.Optional[str] = None,
     sparse_checkout: bool = False,
     verbose: bool = True,
     metadata: t.Optional[t.Dict[str, t.Any]] = None,
     model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
 ) -> Tag:
+    from spacy.cli.project.clone import project_clone
```

Comment:
remove project_clone and project_pull, support only importing from local project directory